### DISCUSSION TITLE
A question concerning `TextFlowContainer.TextAnchor`.

### DISCUSSION THREAD

**Discussion by HiddenNode** [ppy/osu-framework#5498] [Q&A] [answered]
I am trying to understand how `TextFlowContainer.TextAnchor` affects layouting.

This is what `TextFlowContainer` looks like (minus the violet background) with:
```
Text = "A paragraph that does not fit into one line inside this TextFlowContainer" 
TextAnchor = Anchor.BottomRight
```
![text](https://user-images.githubusercontent.com/101936124/199285235-9f6b4327-64e2-4188-85d0-5ea0dc623d58.PNG)

Is the text here layed out correctly, to be more specific, the second line is longer than the first line, is this how it should be layed out?

**Comment by smoogipoo** [comment]
Hmm, it could be that this is counter-expectations... Is there any similar construct with HTML/CSS that could be referenced?

**Reply by HiddenNode** [reply]
None that i am aware of, i did some googling and found no similar HTML/CSS construct.

**Reply by peppy** [reply]
This definitely looks like a bug to me.

**Comment by peppy** [comment, marked as answer]
Tracking via https://github.com/ppy/osu-framework/issues/5499.
